pkg/operator/api/api.go

```diff
@@ -1,4 +1,6 @@
 /*
+Package api for the operator api manager.
```

Comment:
No, the only way I can get the error to go away is if this is the first line of comments. Many of the errors like this are really pedantic and makes me wonder if it would be better not to require being golint clean.